### ISSUE TITLE
Crash right after choosing to backup software selection

### ISSUE THREAD

**Issue by ilsh1964** [linuxmint/mintbackup#45]
mintbacup version: 2.28 (new dev)

$ mintbackup 
** (mintbackup.py:30156): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-XaKXwwfo17: Connection refused
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.

(mintbackup.py:30156): GLib-GIO-ERROR **: Settings schema 'com.linuxmint.install' is not installed
Trace/breakpoint trap

**Comment by clefebvre** [comment]
Need to install the latest mintinstall.
